cmds.go

```diff
@@ -198,22 +198,15 @@ func checkEnvironment(socket, certPath string) bool {
 }
 
 func printExport(socket, certPath string) {
-	for name, value := range exports(socket, certPath) {
-		switch filepath.Base(os.Getenv("SHELL")) {
-		case "fish":
-			if value == "" {
-				fmt.Printf("    set -e %s\n", name)
-			} else {
-				fmt.Printf("    set -x %s %s\n", name, value)
-			}
-		default: // default command to export variables POSIX shells, like bash, zsh, etc.
-			if value == "" {
-				fmt.Printf("    unset %s\n", name)
-			} else {
-				fmt.Printf("    export %s=%s\n", name, value)
-			}
-		}
-	}
+	env := exports(socket, certPath)
```

Comment:
The downside to this is that if another env var is introduced tomorrow, this change will no longer report it, as it hard-codes the env var keys.  Your call, though, Sven.

nit: looks like 201/202 were not both formatted by gofmt.

Otherwise, LGTM.